### PR TITLE
Guf installer configs

### PR DESCRIPTION
**Pull request by stefanomelchior**
This should complete PR #21

### CODE REVIEW

**Comment by Flohack74** [comment]
Did you know that you can always add/edit/delete files from a PR while its open? Simply push on the same branch again more commits ^^ I would prefer to have everything in one place.

**Comment by Flohack74** [comment]
Also, index.json was changed by someone else and nneeds conflict resolution, so you need to first pull master branch into your branch, resolve and then push again

**Review by mariogrip — APPROVED** [review]
LGTM :+1: once device config is merged, this can be merged

**Pull request merged**
